manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml

```diff
@@ -555,7 +555,7 @@ spec:
                   "@type": "type.googleapis.com/google.protobuf.StringValue"
                   value: |
                     {{- if not .Values.telemetry.v2.stackdriver.configOverride }}
-                    {"enable_mesh_edges_reporting": {{ .Values.telemetry.v2.stackdriver.topology }}, "disable_server_access_logging": {{ not .Values.telemetry.v2.stackdriver.logging }}, "meshEdgesReportingDuration": "600s"}
+                    {"enable_mesh_edges_reporting": {{ .Values.telemetry.v2.stackdriver.topology }}, "access_logging": {{ .Values.telemetry.v2.stackdriver.outboundAccessLogging }}, "meshEdgesReportingDuration": "600s"}
```

Comment:
This is a breaking change. We should make it an either new option or old option, and then add a deprecation warning within istioctl about this field.

Reply:
Having old option in outbound filter didn't had any effect before as we didn't have any client logs before.. adding the old option back to inbound filters... please see here: https://github.com/istio/istio/pull/26446/files

Where do we add warning for istioctl? Proto field has been marked as deprecated..